src/Pages/AdminTaskTable/components/EmptyTableState.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  EmptyStateBody,
+  EmptyState,
+  EmptyStateVariant,
+  Title,
+  EmptyStateIcon,
+  Button,
+} from '@patternfly/react-core';
+import { SearchIcon, PlusCircleIcon } from '@patternfly/react-icons/dist/js/icons';
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
+  emptyStateContainer: {
+    display: 'flex',
+    flexGrow: 1,
+  },
+  emptyStateBody: {
+    marginBottom: '16px',
+  },
+});
+
+interface Props {
+  notFiltered?: boolean;
+  clearFilters: () => void;
+}
+
+const EmptyTableState = ({ notFiltered, clearFilters }: Props) => {
+  const classes = useStyles();
+  return (
+    <EmptyState variant={EmptyStateVariant.full} className={classes.emptyStateContainer}>
+      <EmptyStateIcon icon={notFiltered ? PlusCircleIcon : SearchIcon} />
+      <Title headingLevel='h2' size='lg' ouiaId='empty_custom_title'>
+        {notFiltered ? 'No tasks' : 'No tasks match the filter criteria'}
```

Comment:
hmm we may only need page name: ie "tasks" "repositories" to unify this component.